### PR TITLE
CT-1881 - Make ICO decision file mandatory when closing a case

### PR DESCRIPTION
**Pull request by stephenrichards**

### CODE REVIEW

File: spec/factories/case/ico_sars.rb [hunk 1]
```diff
@@ -162,36 +162,37 @@
...
+    #
+    #
+    # after(:create) do |kase, evaluator|./spec/services/case_finder_service_spec.rb:365
+    #   kase.attachments.push evaluator.ico_decision_files
+    #   kase.save!
+    #   create :case_transition_close_ico,
+    #          case_id: kase.id
+    #   kase.reload
+    # end
+  # end
```
Comment (aliuk2012) [inline spec/factories/case/ico_sars.rb]: Why is this factory commented out?

File: README.md [hunk 2]
```diff
@@ -3,7 +3,8 @@
 [![Code Climate](https://codeclimate.com/github/ministryofjustice/correspondence_tool_staff/badges/gpa.svg)](https://codeclimate.com/github/ministryofjustice/correspondence_tool_staff)
 [![Test Coverage](https://codeclimate.com/github/ministryofjustice/correspondence_tool_staff/badges/coverage.svg)](https://codeclimate.com/github/ministryofjustice/correspondence_tool_staff/coverage)
 [![Issue Count](https://codeclimate.com/github/ministryofjustice/correspondence_tool_staff/badges/issue_count.svg)](https://codeclimate.com/github/ministryofjustice/correspondence_tool_staff)
-[![Build Status](https://semaphoreci.com/api/v1/aliuk2012/correspondence_tool_staff/branches/ct-1778-ico-filter/badge.svg)](https://semaphoreci.com/aliuk2012/correspondence_tool_staff)
+[![Build Status](https://semaphoreci.com/api/v1/aliuk2012/correspondence_tool_staff/branches/ct-1847-overturned-visibility/badge.svg)](https://semaphoreci.com/aliuk2012/correspondence_tool_staff)
```
Comment (aliuk2012) [inline README.md]: Not sure why this badge is being updated nor why it was pointing to the wrong branch this line should be `[Build Status](https://semaphoreci.com/aliuk2012/correspondence_tool_staff)` so its based on the master branch

**Review by aliuk2012 — CHANGES_REQUESTED** [review]

File: spec/factories/case/ico_sars.rb (hunk 1 above)
Comment (stephenrichards) [reply spec/factories/case/ico_sars.rb]: Because it's a duplicate after merge with master - I'll delete it.

File: README.md (hunk 2 above)
Comment (stephenrichards) [reply README.md]: I just copied what semafore said to copy

**Review by aliuk2012 — DISMISSED** [review]

**Review by beccapearce — APPROVED** [review]

**Pull request merged**